usr/share/device-quirks/scripts/VENDORS

```diff
@@ -0,0 +1,10 @@
+AOKZOE::aokzie
```

Comment:
typo

Reply:
I will have to go through all of it to catch these minor items like typos once I am done converting over the last two remaining manufacture scripts, then on to testing each script for each vendor/model in a VM just to make sure they are working correctly. They should as I have been testing each separately, but you never know.